ValveResourceFormat/IO/ModelExtract.Mesh.cs

```diff
@@ -325,7 +325,7 @@ private static void FillDatamodelVertexData(VBIB.OnDiskBufferData vertexBuffer,
 
                 var material = drawCall.GetProperty<string>("m_material") ?? drawCall.GetProperty<string>("m_pMaterial");
 
-                if (material != null && materialInputSignature.Elements.Length == 0 && materialInputSignatures != null)
+                if (material != null && materialInputSignatures != null && (materialInputSignature.Elements == null || materialInputSignature.Elements.Length == 0))
```

Comment:
Elements itself can't be null I think. Only zero length.

Reply:
I don't know why but, I was getting an error at this point and it was breaking all the models
![1](https://github.com/ValveResourceFormat/ValveResourceFormat/assets/55619735/b800956b-e40d-4b10-9888-fa29ea244f5a)
 after fixing it, I got this:
![2](https://github.com/ValveResourceFormat/ValveResourceFormat/assets/55619735/8ba7b082-0726-4126-aa76-bfd11db8736c)

Reply:
Okay its likely because struct `default` doesn't initialize the properties.